<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="2" passed="0">
  <reporter-output>
    <line>
      <![CDATA[<br>Verification Failure : expected [abc] but found [xyz]<br>]]>
    </line>
    <line>
      <![CDATA[<a target ="_blank" href=Thu_Nov_11_01_06_19_IST_2021.jpg><img src=Thu_Nov_11_01_06_19_IST_2021.jpg height=300 width=300></img></a>]]>
    </line>
    <line>
      <![CDATA[<br>]]>
    </line>
    <line>
      <![CDATA[<br>]]>
    </line>
    <line>
      <![CDATA[<a target ="_blank" href=Thu_Nov_11_01_06_23_IST_2021.jpg>Screenshot</a>]]>
    </line>
    <line>
      <![CDATA[<br>]]>
    </line>
    <line>
      <![CDATA[<a target ="_blank" href=Thu_Nov_11_01_06_23_IST_2021.jpg><img src=Thu_Nov_11_01_06_23_IST_2021.jpg height=300 width=300></img></a>]]>
    </line>
    <line>
      <![CDATA[<a target ="_blank" href=Thu_Nov_11_01_06_31_IST_2021.jpg>Screenshot</a>]]>
    </line>
    <line>
      <![CDATA[<br>]]>
    </line>
    <line>
      <![CDATA[<a target ="_blank" href=Thu_Nov_11_01_06_31_IST_2021.jpg><img src=Thu_Nov_11_01_06_31_IST_2021.jpg height=300 width=300></img></a>]]>
    </line>
  </reporter-output>
  <suite name="Data Driven Basics" duration-ms="12037" started-at="2021-11-11T01:06:19 IST" finished-at="2021-11-11T01:06:31 IST">
    <groups>
    </groups>
    <test name="Bank Manager Login Test" duration-ms="4835" started-at="2021-11-11T01:06:19 IST" finished-at="2021-11-11T01:06:24 IST">
      <class name="com.w2a.testcases.BankManagerLoginTest">
        <test-method status="FAIL" signature="loginAsBankManager()[pri:0, instance:com.w2a.testcases.BankManagerLoginTest@57d7f8ca]" name="loginAsBankManager" duration-ms="4307" started-at="2021-11-11T01:06:19 IST" finished-at="2021-11-11T01:06:23 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Login not Successfull]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Login not Successfull
at org.testng.Assert.fail(Assert.java:97)
at com.w2a.testcases.BankManagerLoginTest.loginAsBankManager(BankManagerLoginTest.java:27)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:483)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at org.testng.TestRunner$$Lambda$248/2075568954.accept(Unknown Source)
at java.util.ArrayList.forEach(ArrayList.java:1234)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[<br>Verification Failure : expected [abc] but found [xyz]<br>]]>
            </line>
            <line>
              <![CDATA[<a target ="_blank" href=Thu_Nov_11_01_06_19_IST_2021.jpg><img src=Thu_Nov_11_01_06_19_IST_2021.jpg height=300 width=300></img></a>]]>
            </line>
            <line>
              <![CDATA[<br>]]>
            </line>
            <line>
              <![CDATA[<br>]]>
            </line>
            <line>
              <![CDATA[<a target ="_blank" href=Thu_Nov_11_01_06_23_IST_2021.jpg>Screenshot</a>]]>
            </line>
            <line>
              <![CDATA[<br>]]>
            </line>
            <line>
              <![CDATA[<a target ="_blank" href=Thu_Nov_11_01_06_23_IST_2021.jpg><img src=Thu_Nov_11_01_06_23_IST_2021.jpg height=300 width=300></img></a>]]>
            </line>
          </reporter-output>
        </test-method> <!-- loginAsBankManager -->
      </class> <!-- com.w2a.testcases.BankManagerLoginTest -->
    </test> <!-- Bank Manager Login Test -->
    <test name="Add Customer Test" duration-ms="7177" started-at="2021-11-11T01:06:24 IST" finished-at="2021-11-11T01:06:31 IST">
      <class name="com.w2a.testcases.AddCustomerTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.w2a.testcases.AddCustomerTest@2f217633]" name="setUp" is-config="true" duration-ms="6573" started-at="2021-11-11T01:06:12 IST" finished-at="2021-11-11T01:06:19 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="addCustomerTest(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.w2a.testcases.AddCustomerTest@2f217633]" name="addCustomerTest" duration-ms="6942" started-at="2021-11-11T01:06:24 IST" data-provider="dp" finished-at="2021-11-11T01:06:31 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[Mukesh]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ambani]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[411223.0]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Customer added successfully]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Customer Not Added Successfully  !!!]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Customer Not Added Successfully  !!!
at org.testng.Assert.fail(Assert.java:97)
at com.w2a.testcases.AddCustomerTest.addCustomerTest(AddCustomerTest.java:31)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:483)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at org.testng.TestRunner$$Lambda$248/2075568954.accept(Unknown Source)
at java.util.ArrayList.forEach(ArrayList.java:1234)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[<a target ="_blank" href=Thu_Nov_11_01_06_31_IST_2021.jpg>Screenshot</a>]]>
            </line>
            <line>
              <![CDATA[<br>]]>
            </line>
            <line>
              <![CDATA[<a target ="_blank" href=Thu_Nov_11_01_06_31_IST_2021.jpg><img src=Thu_Nov_11_01_06_31_IST_2021.jpg height=300 width=300></img></a>]]>
            </line>
          </reporter-output>
        </test-method> <!-- addCustomerTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.w2a.testcases.AddCustomerTest@2f217633]" name="tearDown" is-config="true" duration-ms="879" started-at="2021-11-11T01:06:31 IST" finished-at="2021-11-11T01:06:32 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.w2a.testcases.AddCustomerTest -->
    </test> <!-- Add Customer Test -->
  </suite> <!-- Data Driven Basics -->
</testng-results>
